README.md

```diff
@@ -20,35 +20,24 @@ Create MDX files showcasing your code and Docz turns them into a live-reloading,
 
 ![docz example](https://cdn-std.dprcdn.net/files/acc_649651/S2YCID)
 
-## Table of Contents
-
-- [Why?](#why)
-- [Getting started](#getting-started)
-- [Build](#build)
-- [Deploy](#deploy)
-- [Examples](#examples)
-- [More info on docz.site](#more-info-on-doczsite)
-- [Used by](#used-by)
-- [Contributors](#contributors)
-- [Contributing](#contributing)
```

Comment:
I think this is helpful for other people navigating through the content here, you might drop off the title here instead, what you think?